doc/developer-guides/hld/hv-vcat.rst

```diff
@@ -0,0 +1,141 @@
+.. _hv_vcat:
+
+Enable vCAT
+###########
+
+vCAT refers to the virtualization of Cache Allocation Technology (CAT), one of the
+RDT (Resource Director Technology) technologies.
+
+ACRN vCAT is built on top of ACRN RDT: ACRN RDT provides a number of physical CAT resources
+(CLOS IDs + cache ways), ACRN vCAT exposes some number of virtual CAT resources to VMs and
+then transparently map them to the assigned physical CAT resources in the ACRN hypervisor;
+VM can take advantage of vCAT to prioritize and partition virtual cache ways for its own tasks.
+
+In current CAT implementation, one CLOS ID corresponds to one IA32_type_MASK_n (type: L2 or L3,
+n ranges from 0 to MAX_CACHE_CLOS_NUM_ENTRIES - 1) MSR and a bit in a capacity bitmask (CBM)
+corresponds to one cache way.
+
+On current generation systems, normally L3 cache is shared by all CPUs on the same socket and
+L2 cache is generally just shared by the hyperthreads on a core. But when dealing with ACRN
+vCAT CLOS IDs assignment, it is currently assumed that all the L2/L3 caches (and therefore all CLOS IDs)
+are system-wide caches shared by all cores in the system, this is done for convenience and to simplify
+the vCAT configuration process. If vCAT is enabled for a VM (abbreviated as vCAT VM), there should not
+be any CLOS ID overlap between a vCAT VM and any other VMs. e.g. the vCAT VM has exclusive use of the
+assigned CLOS IDs.
+When assigning cache ways, however, the VM can be given exclusive, shared, or mixed access to the cache
+ways depending on particular performance needs. For example, use dedicated cache ways for RTVM, and use
+shared cache ways between low priority VMs.
+
+In ACRN, vCAT can be configured via the "VM-Configuration". The vCAT cache ways allocated
```

Comment:
Are you referring to the ACRN Configuration Tool?

Reply:
yes, referring to the vcat configuration